resources/qml/Dialogs/RenameDialog.qml

```diff
@@ -15,17 +15,21 @@ UM.Dialog
     buttonSpacing: UM.Theme.getSize("default_margin").width
 
     property string object: ""
+    property string objectPlaceholder: ""
 
     property alias newName: nameField.text
     property bool validName: true
     property string validationError
     property string dialogTitle: catalog.i18nc("@title:window", "Rename")
     property string explanation: catalog.i18nc("@info", "Please provide a new name.")
+    property string okButtonText: catalog.i18nc("@action:button", "OK")
+
+    property list<Item> extraInfo
```

Comment:
I think this should be an alias instead of a property.

Reply:
Tried that. You get into a bit of trouble because of the whole 'rightButtons' being a group thing that is also an alias into the parent.